experiments/extract_difference_image.py

```diff
@@ -0,0 +1,45 @@
+from PIL import Image
```

Comment:
Please add brief module documentation and example usage.

Reply:
Maybe I should remove this file? already implemented same function in vision.py